staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -81,7 +82,7 @@ type StartFunction func(ctx context.Context, hashValue uint64) (execute bool, af
 type RequestDigest struct {
 	RequestInfo *request.RequestInfo
 	User        user.Info
-	Width       uint
+	Width       fcrequest.Width
```

Comment:
The first commit isn't controversial at all - let's just split it to the separate PR (well - let's just not add additionalLatency field to the Width type but leave everything else as is) and let's just merge it.

Reply:
@wojtek-t  sounds good, i just opened https://github.com/kubernetes/kubernetes/pull/103504

Reply:
I objected to the first commit in the first sentence of https://github.com/kubernetes/kubernetes/pull/103240#issuecomment-869802687

Reply:
But that commit doesn't add additional latency (i.e. the one that I approved didn't add it).

Reply:
A struct named "Width" that only has one field seems a bit pointless.  (more extensive commentary elsewhere.)